guides/common/modules/proc_configuring-the-discovery-service.adoc

```diff
@@ -37,27 +37,37 @@ endif::[]
 hosts or virtual machines can be created with incorrect network configurations. On Dell servers, the new naming scheme can be disabled via `biosdevname=1` kernel command line option, for other hardware or virtual machines, the new naming scheme can be completely turned off via `net.ifnames=1`.
 endif::[]
 
-To use {ProjectServer} to provide the Discovery image, install the following RPM packages:
-
-* `tfm-rubygem-foreman_discovery`
-ifdef::satellite[]
-* `foreman-discovery-image`
+.Procedure
+. To use {ProjectServer} to provide the Discovery image, run the following commands:
```

Comment:
It only lists a single installer command, so singular:
```suggestion
. To use {ProjectServer} to provide the Discovery image, run the following command:
```

I'd also say that it does more than provide the discovery image. It installs the Foreman Discovery plugin, which is required if you want to use the Foreman Proxy Discovery plugin from some Foreman Proxy. Then it also installs the Foreman Proxy Discovery Plugin, which technically you don't need on `{ProjectServer}` if you have separate Foreman Proxies, but our instructions typically say that because we don't assume separate Foreman Proxies.

So I'd rephrase it to:
```suggestion
. Install Discovery on {ProjectServer}:
```

However, feel free to come up with some alternative here.

Reply:
Addressed :+1: